ANSWERS.md

```diff
@@ -1,6 +1,11 @@
 ### Questions:
-1. What is the DOM?
-2. What is an event?
-3. What is an event listener?
+1. What is the DOM? Document Object Model that renders the code to be viewed on a browser.
+
+2. What is an event? Event is a specific action that will set off code based on what that action has been defined to be.
+
+3. What is an event listener? EventListener tells the browser to wait/listen for the specified event to occur before taking action.
+
 4. Why would we convert a NodeList into an Array?
-5. What is a component? 
+Because a NodeList does not have a lot of methods on it's own so we convert it into an Array in order to get access to more methods and properties.
+
+5. What is a component? A component is a section of a website that involves HTML, CSS & JavaScript together to function properly. This components code can be reused as they are all tied in together. A component can also save time and memory by requiring the code to only be written once to function.
```

Comment:
Please be sure to resolve any merge conflict issues if working in another branch at any time. Very much would've like to further review your comprehension answers in their entirety.